src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -234,9 +234,8 @@ public string KeychainSecurityGroup
         /// <param name="resource">Identifier of the target resource that is the recipient of the requested token.</param>
         /// <param name="clientId">Identifier of the client requesting the token.</param>
         /// <returns>It contains Device Code, its expiration time, User Code.</returns>
-#if iOS
-        [Obsolete("This device profile API should only be used on text-only devices, and not on this target platform which is offering an interactive authentication experience. For details please see https://aka.ms/AdalNetConfFlows")]
-#endif
+        [Obsolete("Microsoft.IdentityModel.Clients.ActiveDirectory has been deprecated and replaced with Microsoft.Identity.Client. " +
+            "To migrate an app using Device Code flow, see https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-net-migration-public-client?tabs=devicecode")]
```

Comment:
why not show as an error instead?

Reply:
this would make the NuGet package un-usable